### PR TITLE
Prep Javascript Dom Creation First Draft W.I.P Nelson Daza

### PR DESCRIPTION
**Pull request by neldaza**
squashed out every other step, got stuck on final question, any tips?
Screen Shot 2021-09-25 at 3 53 56 PM
s?

### CODE REVIEW

File: prep-javascript-dom-creation/index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  <title>JavaScript DOM Creation</title>
+</head>
+
+<body>
+  <div id="container">
+
+  </div>
+  <script>
+    // Your code here
+    var $container = document.querySelector('div.container');
```
Comment (scbowler) [inline prep-javascript-dom-creation/index.html]: Here the selector should be `#container` or `div#container`

File: prep-javascript-dom-creation/index.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    console.log(container);
+    var $heading = document.createElement('h1');
+    $heading.textContent = 'Hello World';
+    console.log($heading);
+    $heading.appendChild($container);
```
Comment (scbowler) [inline prep-javascript-dom-creation/index.html]: Here you want to append `$heading` into `$container` so it should be the other way around compared to your code. `$container.appendChild($heading);`.

**Review by scbowler — CHANGES_REQUESTED** [review]
A couple updates needed. Let me know if you have any questions.

**Comment by neldaza** [comment]
updated! thanks Scott!
Screen Shot 2021-09-27 at 7 28 55 PM
Screen Shot 2021-09-27 at 7 27 30 PM

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**